logic/path/src/path.rs

```diff
@@ -678,7 +686,8 @@ mod tests {
 
         // path: 0 -> 1 -> 2 -> 3 -> 4
         let tp = cp
-            .to_path(&resolver, addrs[4])
+            .clone()
```

Comment:
is that better than the previous version?

Reply:
This is a test.